### PR TITLE
Release/3.2.2

### PR DESCRIPTION
**Pull request by hdbeukel**
A tiny new release to standardize the minimal Java version specified (system requirements in DESCRIPTION). Along the way, some other small changes were made:
- Replaced old Travis CI workflow with updated GitHub actions workflows, running checks on more platforms and R versions
- Used `revdepcheck` to check reverse dependencies (currently one package: `EvaluateCore`).
- Regenerated docs using latest Roxygen version

### CODE REVIEW

**Comment by hdbeukel** [comment]
CRAN submission is pending, leave PR open until successful release has been confirmed.

File: DESCRIPTION [hunk 1]
```diff
@@ -16,9 +16,9 @@ Description: Core Hunter is a tool to sample diverse, representative subsets fro
     <http://www.corehunter.org>).
 Depends: R (>= 3.2.3), rJava (>= 0.9-8)
 Imports: naturalsort (>= 0.1.2), methods
-SystemRequirements: Java JRE 8 or higher
+SystemRequirements: Java (>= 8)
```
Comment (hdbeukel) [inline DESCRIPTION:19]: @daveneti this is the actual tiny change for which the release was made, after receiving the email below from Prof. Brian Ripley.

---

Most packages have

SystemRequirements: Java (>= 1.8)

and 'Writing R Extensions' asks you not to use '1.8', and not to specify 
a JDK.  But we see

AMAPVox/DESCRIPTION:SystemRequirements: Java 1.8 64-Bit (Oracle or Corretto)
DatabaseConnector/DESCRIPTION:SystemRequirements: Java version 8 or 
higher (https://www.java.com/)
EvidenceSynthesis/DESCRIPTION:SystemRequirements: Java version 8 or 
higher (https://www.java.com/)
JGR/DESCRIPTION:SystemRequirements: Java JDK 1.8 or higher
PortfolioEffectEstim/DESCRIPTION:SystemRequirements: Java (>= 1.7)
PortfolioEffectHFT/DESCRIPTION:SystemRequirements: Java (>= 1.7)
RJDemetra/DESCRIPTION:SystemRequirements: Java JRE 8 or higher.
Rbgs/DESCRIPTION:SystemRequirements: Java JRE 1.6 or higher ; 
Xuggle-xuggler-5.4.jar in
SqlRender/DESCRIPTION:SystemRequirements: Java version 8 or higher 
(https://www.java.com/
corehunter/DESCRIPTION:SystemRequirements: Java JRE 8 or higher
ggdemetra/DESCRIPTION:SystemRequirements: Java JRE 8 or higher.
iMRMC/DESCRIPTION:SystemRequirements: Java JDK 1.7 or higher
nlrx/DESCRIPTION:SystemRequirements: NetLogo (>= 5.3.1), Java (JDK 1.8), 
pandoc,
rcdk/DESCRIPTION:SystemRequirements: Java JDK 8 or higher
rjdmarkdown/DESCRIPTION:SystemRequirements: Java SE 8 or higher
rkafka/DESCRIPTION:SystemRequirements: Java JDK 1.7 or higher,Apache 
Kafka 2.8.0-0.8.1.1
rviewgraph/DESCRIPTION:SystemRequirements: 'Java' version >= 8
scatteR/DESCRIPTION:SystemRequirements: Java 8 or higher
staplr/DESCRIPTION:SystemRequirements: Java 8 or higher


Please change to the standard form before 2023-05-23 to safely retain 
your package on CRAN.

**Review by ghost — APPROVED** [review]
I have not tested it, but I see no major problems.

**Comment by hdbeukel** [comment]
There were some notes and 1 error in the test runs on CRAN submission, so it has been refused. I will further investigate and make the necessary adjustments before merging.

**Comment by hdbeukel** [comment]
Made some changes in an attempt to fix CRAN submission issues. New submission is pending.

**Comment by hermandebeukelaer** [comment]
@daveneti After a few attempts I finally got the new release accepted by CRAN. Do you still approve the latest version, including the minor changes I made to get through CRAN checks?

**Comment by daveneti** [comment]
Yes, still approve :)

**Pull request merged**